### PR TITLE
add omitWhenMerged=true to publisher field in release-schema

### PR DESCRIPTION
**Pull request by odscjen**
adding omitWhenMerged = true to release schema for 1.2 as discussed in #1483

### CODE REVIEW

**Review by duncandewhurst — CHANGES_REQUESTED** [review]
The schema changes look good. Please can you update the [changelog](https://github.com/open-contracting/standard/blob/1483-publisher-omitWhenMerged/docs/history/changelog.md).

FYI - you don't need to set labels on PRs, we only use those for categorising issues. Also, if you [use a keyword to link a PR to an issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) it will automatically close the issue once the PR is merged, e.g. Closes #1483

**Comment by odscjen** [comment]
Great, I'll do that. Can I just check how the changelog works, am I right in assuming that I'd be adding a line explaining this under the section [1.2] - YYY-MM-DD / schema / clarifying merging behavior? And that I'd add it to the bottom of that section?

**Comment by duncandewhurst** [comment]
Since this is just fixing an omission in a field that is added in the same release, I think it's fine just to update https://github.com/open-contracting/standard/blob/686e1b7229f9c68c5cc8ff7a689f59278941346b/docs/history/changelog.md?plain=1#L170 to read:

```markdown
* [#1376](https://github.com/open-contracting/standard/pull/1376), [#1489](https://github.com/open-contracting/standard/pull/1489) `publisher`
```

**Review by duncandewhurst — APPROVED** [review]

File: docs/history/changelog.md [hunk 1]
```diff
@@ -167,7 +167,7 @@ Per the [normative and non-normative content and changes policy](https://docs.go
   * [#1165](https://github.com/open-contracting/standard/pull/1165) `statusDetails` to `Tender`, `Award` and `Contract`
   * [#1455](https://github.com/open-contracting/standard/pull/1455) `Milestone.dueAfterDate`
   * [#1326](https://github.com/open-contracting/standard/pull/1326) [#1431](https://github.com/open-contracting/standard/pull/1431) `links`
-  * [#1376](https://github.com/open-contracting/standard/pull/1376) `publisher`
+  * [#1376](https://github.com/open-contracting/standard/pull/1376),[#1489](https://github.com/open-contracting/standard/pull/1489) `publisher`
```
Comment (jpmckinney) [inline docs/history/changelog.md]:
```suggestion
  * [#1376](https://github.com/open-contracting/standard/pull/1376), [#1489](https://github.com/open-contracting/standard/pull/1489) `publisher`
```

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**